Docs/Profiling.md

```diff
@@ -0,0 +1,80 @@
+| Frame      | Frame time represents the total amount of time spent generating one frame of the app. Since both the Game and Draw threads sync up before finishing a frame, frame time is often close to the time being shown in one of these threads. |
+| Game       | If Frame time is close to Game time, the app's performance is likely being bottlenecked (negatively impacted) by the game thread. The game thread is resposible for most app logic, especially Blueprint logic.                             |
+| Draw       | If Frame time is close to Draw time, the game's performance is likely being bottlenecked by the rendering thread. The rendering thread is responsible for decideding what to render and submitting work to the GPU.                         |
+| GPU        | GPU time measures how long the video card took to render the scene. Since GPU time is synced to the frame, it will likely be similar to Frame time.                                                                                        |
+| Draw Calls | Draw calls can be thought of as the number on times a graphics API (such as [DirectX](https://en.wikipedia.org/wiki/DirectX)) is told to render an object.                                                                                  |
+| Polys      | Represents the number of polygons which are currently being submitted to the graphics API for rendering. This number may vary slightly to actual number being rendered due to frustum clipping or geometry generation on the GPU.                    |
+
+> [!NOTE]
+> It is particularly important to utilize the visual profiler to track frame time when running on the device as opposed to running in editor or an emulator. The most accurate performance results will be depicted when running on the device with "Shipping" build configuration.
+
+## Example level
+
+To better understand the `GTVisualProfiler` look at the `\GraphicsToolsProject\Plugins\GraphicsToolsExamples\Content\Profiling\Profiling.umap` level.
+
+## General recommendations
+
+Performance can be an ambiguous and constantly changing challenge for mixed reality developers and the spectrum of knowledge to rationalize performance is vast. There are some general recommendations for understanding how to approach performance for an application though.
+
+It is useful to simplify the execution of an application into the pieces that run on the *CPU* or the *GPU* and thus identify whether an app is bounded by either component. There can be bottlenecks that span both processing units and some unique scenarios that have to be carefully investigated. However, for getting started, it is good to grasp where an application is executing for the most amount of time.
+
+### GPU bounded
+
+If the longest bar on the profiler is the "GPU" time then your app is likely GPU bound.
```

Comment:
>If the longest bar on the profiler is the "GPU" time then your app is likely GPU bound [](start = 0, length = 86)

This will tend to always be the longest bar in the profiler because it is synced to the frame, no? Is there any use in having this one if it tracks closely the frame one?

Reply:
I agree it does seem somewhat redundant. But, I would like to keep it around since the engine tracks the time independently and may prove useful on other platforms (or maybe Epic will eventually subtract the vsync idle time... which would be nice).